src/lib/validation.ts

```diff
@@ -4,7 +4,7 @@ import Place from '../services/place';
 import RemotePlaceResolver from './remote-place-resolver';
 import { RemotePlace } from './cht-api';
 
-import ValidatorDateOfBirth from './validator-dob';
+import ValidatorDob from './validator-dob';
```

Comment:
```suggestion
import ValidatorDoB from './validator-dob';
```

or DateOfBirth? Dunno how common that acronym is...?